### DISCUSSION TITLE
Docker images deprecated?

### DISCUSSION THREAD

**Discussion by edgimar** [sagemathinc/cocalc#7414] [General]
I noticed that the cocalc-docker repository is archived and the documentation indicates that it is deprecated.  Is there a replacement for this, for users who are depending on docker images currently?

**Comment by williamstein** [comment]
No.  We have to significantly tighten our focus for the time being due to lack of resources.

**Comment by edgimar** [comment]
Would it be reasonable if others in the community chose to maintain an independent fork of the repo?  Perhaps this could be a good way forward if there are people sufficiently interested in maintaining it.

**Comment by KenwoodFox** [comment]
Yeah im still using the docker version and everything works fine, for local robotics teams and individual people nothing seems wrong. I wouldn't mind stepping up just to help keep things ticking along.

**Comment by mforbes** [comment]
Where did the archived repo go?  We are still using it too and need to refer to some older issues until we switch to On Prem (if we can make that work with our security model).

**Comment by edgimar** [comment]
If someone has an up-to-date version of the repo, maybe they could push it back to github (as a source repo), and we can continue to use and update it?

**Comment by KenwoodFox** [comment]
Are there any alternatives? I'm on a robotics team and everything was working fine till our lighthouse container saw the repos go offline, what are we gonna do moving forward? just leave the latest in the repo pls!

**Reply by haraldschilly** [reply]
> Are there any alternatives? 

There is a simplified variant of the production system for a kubernetes cluster: https://onprem.cocalc.com/

That's more secure, robust, isolates projects (resource management), is scalable, and has more functionality.

**Reply by edgimar** [reply]
And it appears to have one other fundamental difference: cost.  Is this correct?

**Comment by williamstein** [comment]
The docker repos are still available:
https://hub.docker.com/r/sagemathinc/cocalc-docker
